### PR TITLE
Fixed sql_create_index for mysql

### PR DESCRIPTION
**Pull request by dakusan**
This needs to use "alter table" instead of "create index" due to this 6 year old bug in mysql:
https://bugs.mysql.com/bug.php?id=48875

### CODE REVIEW

File: django/db/backends/mysql/schema.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_create_pk = "ALTER TABLE %(table)s ADD CONSTRAINT %(name)s PRIMARY KEY (%(columns)s)"
     sql_delete_pk = "ALTER TABLE %(table)s DROP PRIMARY KEY"
+
```
Comment (charettes) [inline django/db/backends/mysql/schema.py]: You've got trailing whitespace on this line.

**Comment by charettes** [comment]
This will require a ticket and test that makes sure the case of the table name is preserved on index creation. MySQL will never stop to amaze me.

**Comment by dakusan** [comment]
Oops, sorry about the whitespace. This is actually something that has been bugging me for quite a while regarding github. Is there a way to easily fix that mistake I made, or do I need to edit the file again and make a separate pull request?

**Comment by charettes** [comment]
From the branch name I suppose you used the Web GUI to submit this change. If you want to issue another commit on this branch I think you'll have to check it out locally first.

**Comment by timgraham** [comment]
Duplicated by https://github.com/django/django/pull/6119 (still missing a test though).

**Pull request closed without merging**